install/kubernetes/cilium/values.yaml

```
@@ -1609,9 +1609,11 @@ nodeinit:
   securityContext:
     privileged: true
 
+  # -- bootstrapDir is the location of the directory containing the bootstrapFile
+  bootstrapDir: "/tmp/cilium-bootstrap.d"
   # -- bootstrapFile is the location of the file where the bootstrap timestamp is
   # written by the node-init DaemonSet
```

This PR introduce a change in how this value is used, so some clarification would be welcome here
```suggestion
  # -- bootstrapFile is the file name in bootstrapDir where the bootstrap timestamp is
  # written by the node-init DaemonSet
```

---

Not anymore with the use of `dir` and `base`